clr-k8s-examples/create_stack.sh

```diff
@@ -30,7 +30,9 @@ METALLB_VER="${CLRK8S_METALLB_VER:-v0.8.1}"
 NPD_VER="${CLRK8S_NPD_VER:-v0.6.6}"
 PROMETHEUS_VER="${CLRK8S_PROMETHEUS_VER:-f458e85e5d7675f7bc253072e1b4c8892b51af0f}"
 CNI=${CLRK8S_CNI:-"canal"}
+if [[ ! -z "$RUNNER" ]]; then RUNNER="${RUNNER}"; fi
```

Comment:
This looks benign? I read it as 'if runner is non zero set runner to runner. Am I missing some subtlety?

Reply:
This is used in case for Cilium, since cilium is using Runner env variable, so if runner is not declared, it uses the Runner as set by setup_system.sh